src/controllers/expenses.controllers.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable space-before-function-paren */
+/* eslint-disable no-console */
+'use strict';
+
+const expensesServices = require('../services/expenses.services');
+const usersServices = require('../services/users.services');
+
+const get = async (req, res) => {
+  const query = req.query;
+  let expenses = [];
+
+  if (!query) {
+    expenses = await expensesServices.getAllExpenses();
+
+    res.send(expenses
+      .map(expense => expensesServices.normalize(expense)));
+
+    return;
+  }
+
+  expenses = await expensesServices.getByQueryExpenses(query);
+
+  res.send(expenses
+    .map(expense => expensesServices.normalize(expense)));
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+
+  if (!id) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const expense = await expensesServices.getByIdExpense(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const isUserExist = await usersServices.getByIdUser(userId);
+  const allValueExist = userId
+    && spentAt
+    && title
+    && String(amount)
+    && category
+    && isUserExist;
+
+  if (!allValueExist) {
+    res.sendStatus(400);
+  }
+
+  const expense = await expensesServices.createExpense({
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  });
+
+  res.statusCode = 201;
+  res.send(expensesServices.normalize(expense));
+};
+
+const update = async (req, res) => {
+  const { id } = req.params;
+  const newValues = req.body;
+
+  if (!newValues) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const expense = await expensesServices.getByIdExpense(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  await expensesServices.updateExpense(id, newValues);
+
+  const newExpense = await expensesServices.getByIdExpense(id);
+
+  res.send(expensesServices.normalize(newExpense));
```

Comment:
you can return newExpense from `updateExpense` function and do the following:
```suggestion
  const newExpense = await expensesServices.updateExpense(id, newValues);
  res.send(expensesServices.normalize(newExpense));
```

Reply:
just do not forget to return it from `updateExpense`